docs/source/notes/modules.rst

```diff
@@ -0,0 +1,433 @@
+.. _modules:
+
+Modules
+=======
+
+PyTorch uses modules to both build and represent neural networks. Modules are:
+
+* **Composable building blocks of stateful computation.**
```

Comment:
what does composable mean here?

Reply:
The standard definition (from Wikipedia): "A highly composable system provides components that can be selected and assembled in various combinations to satisfy specific user requirements."

Would you recommend removing "composable" here? I think we still get the idea across without it.

Reply:
Update: removed "Composable"